tests/production/ProductionTestsUtils.js

```diff
@@ -24,15 +25,26 @@ gpii.tests.productionConfigTesting.config = {
     configPath: "%gpii-universal/tests/configs"
 };
 
+gpii.tests.productionConfigTesting.cloudUrl = url.parse(
+    process.env.GPII_CLOUD_URL
+);
+fluid.log("gpii.tests.productionConfigTesting.cloudUrl = '" + JSON.stringify(gpii.tests.productionConfigTesting.cloudUrl, null, "\t") + "'");
+
+gpii.tests.productionConfigTesting.couchdbUrl = url.parse(
+    process.env.GPII_COUCHDB_URL
+);
+fluid.log("gpii.tests.productionConfigTesting.couchdbUrl = '" + JSON.stringify(gpii.tests.productionConfigTesting.couchdbUrl, null, "\t") + "'");
+
 // Access token deletion requests
 fluid.defaults("gpii.tests.cloud.oauth2.accessTokensDeleteRequests", {
     gradeNames: ["fluid.component"],
     components: {
         getAccessTokensRequest: {
             type: "kettle.test.request.http",
             options: {
-                port: "5984",
-                hostname: "couchdb",
+                port: gpii.tests.productionConfigTesting.couchdbUrl.port,
```

Comment:
Same as above. The base grade with couchdbUrl options probably can be shared by multiple production tests.

Reply:
Okay done.
I also added a similar base `cloudStatusRequest` grade for `CloudStatusProductionTests.js`